lib/rubocop/cop/ezcater/direct_env_checks.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Ezcater
+      # Use the `Rails.configuration.x` namespace for configuration backed by
+      # environment variables, rather than inspecting `ENV` directly.
+      #
+      # Centralizing application configuration helps avoid scattering it
```

Comment:
🎱 "Centralizing ... avoids scattering" reads like a bit of a tautology.

Maybe this line doesn't apply to this cop. Instead something like:

Restricting environment variables references to be within application configuration makes it more obvious which environment variables an applications relies upon.

Reply:
Seems reasonable to me. 👍